rust/tools/authorization-logic/README.md

```diff
@@ -77,11 +77,12 @@ claim, and the recipient uses the public key to check the signature.
 To compile a program called FILENAME stored in INPUT_DIR, run:
 
 ```
-cargo run FILENAME INPUT_DIR OUTPUT_DIR
+cargo run -- --in-dir INPUT_DIR --out-dir OUTPUT_DIR FILENAME
 ```
 
-which will result in a Souffle file called FILENAME.dl in OUTPUT_DIR. If Souffle
-is installed, it will be run on the generated code.
+which will result in a Souffle file called FILENAME.dl in OUTPUT_DIR. INPUT_DIR
+and OUTPUT_DIR are optional, and will use the current working directory if
+unspecified. If Souffle is installed, it will be run on the generated code.
```

Comment:
This might not be the right PR for this, but I think the interface would be cleaner if, instead of the command line arguments specifying a filename, input dir, and output dir, specifying just an input file and an output file. Feel free to defer to an issue or ignore if this is not the right spot for this.

Reply:
Also, should the `skip` option be documented here, or is that too internal to worry about in this file?

Reply:
I like the idea of input file and output file, but it's a little complicated because we also use the output directory for Souffle files. Certainly solvable, but I think I'll wait to do it in another PR.

`skip` seems internal/specialized enough that it's probably not worth highlighting here. It seems simpler to have this be the minimal basic example, and rely on `cargo run -- --help` (or `-h`) for the detailed, living documentation. But with that in mind, I've also added a mention of the help command here.